Dockerfile

```diff
@@ -1,11 +1,18 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM python:3.8-slim-buster
```

Comment:
@anarkrypto could you give more details about the goal of this change and why it's necessary?